.common-ci.yml

```diff
@@ -209,3 +210,18 @@ release:staging-ubi9:
     - .dist-ubi9
   needs:
     - image-ubi9
+
+# Triggers the downstream container-dev/ngc-automation pipeline to publish the image to NGC
+# This stage is triggered specifically on tags   
+trigger-downstream-publishing-pipeline:
```

Comment:
Does this job remove the need for anything that depends on `.release:external:`? (we still have `release:ngc-ubi9`) in `.nvidia.yaml`.